### ISSUE TITLE
latest version breaks nuxt ssr

### ISSUE THREAD

**Issue by mavrick** [ktquez/vue-disqus#18]
importing as a component results in:
```{ TypeError: a.component is not a function
    at install (C:\test\node_modules\vue-disqus\dist\vue-disqus.cjs.js:1:1448)
    at renderAsyncComponent (C:\test\node_modules\vue-server-renderer\build.js:7530:11)
    at RenderContext.renderNode (C:\test\node_modules\vue-server-renderer\build.js:7377:7)
    at RenderContext.next (C:\test\node_modules\vue-server-renderer\build.js:2403:14)
    at cachedWrite (C:\test\node_modules\vue-server-renderer\build.js:2263:9)
    at renderStringNode$1 (C:\test\node_modules\vue-server-renderer\build.js:7560:5)
    at RenderContext.renderNode (C:\test\node_modules\vue-server-renderer\build.js:7369:5)
    at RenderContext.next (C:\test\node_modules\vue-server-renderer\build.js:2403:14)
    at RenderContext.cachedWrite [as write] (C:\test\node_modules\vue-server-renderer\build.js:2263:9)
    at RenderContext.next (C:\test\node_modules\vue-server-renderer\build.js:2406:14)
    at RenderContext.next (C:\test\node_modules\vue-server-renderer\build.js:2412:12)
    at RenderContext.cachedWrite [as write] (C:\test\node_modules\vue-server-renderer\build.js:2263:9)
    at RenderContext.next (C:\test\node_modules\vue-server-renderer\build.js:2406:14)
    at RenderContext.next (C:\test\node_modules\vue-server-renderer\build.js:2412:12)
    at RenderContext.cachedWrite [as write] (C:\test\node_modules\vue-server-renderer\build.js:2263:9)
    at RenderContext.next (C:\test\node_modules\vue-server-renderer\build.js:2406:14) statusCode: 500, name: 'TypeError' }```

i suspect it is this:
```function install(a){a.component('VueDisqus',VueDisqus)}```

importing as a nuxt plugin disabling ssr results in:
```<vue-disqus> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
```

**Comment by ktquez** [comment]
I'll check and resolve, I have not yet tested on Nuxt, but in vue-templates and browser is working.

As soon as I correct, I communicate here and I update in the README the use in Nuxt.

**Comment by ktquez** [comment]
At the moment you can pick up the [v3.0.1](https://github.com/ktquez/vue-disqus/tree/v3.0.1) (last)

To use:
```vue
<template>
  <section class="container">
      <vue-disqus shortname="your_shortname" />
  </section>
</template>

<script>
import VueDisqus from 'vue-disqus/dist/vue-disqus.vue'

export default {
  components: {
    VueDisqus
  }
}
```

But I will soon work on the fix to use vue-disqus as a Nuxt-compatible install plugin

**Comment by ktquez** [comment]
@mavrick Fixed [v3.0.3](https://github.com/ktquez/vue-disqus/tree/v3.0.3)
